src/pages/settings/Payments/AddDebitCardPage.js

```diff
@@ -301,7 +301,7 @@ class DebitCardPage extends Component {
                                     <StatePicker
                                         onChange={value => this.clearErrorAndSetValue('addressState', value)}
                                         value={this.state.addressState}
-                                        hasError={lodashGet(this.state.errors, 'addressState', false)}
+                                        errorText={lodashGet(this.state.errors, 'addressState', {})}
```

Comment:
I'm concerned about what this Picker will recieve. I think this specific Picker will recieve a `boolean` (what `hasError` formerly expected) but we expect a `string`.  Should we change the logic inside Picker's `errorText` to recieve also a `boolean` or make sure a `string` is being passed to this Picker? (may other Pickers have this issue).

Reply:
No need to worry about this Picker now, I just needed to use the Parent Component's getErrorText to pass errorText a string. The error style and message of this Picker is now working as expected.